src/App.js

```diff
@@ -1,41 +1,32 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar } from "./components/index";
+import Mockman from "mockman-js";
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { Routes , Route } from 'react-router-dom';
+import { HomePage , SignIn , SignUp , History, Likes, WatchLater, PlayList } from "./Pages/index";
```

Comment:
you don't need to mention that index at the end, by default it goes to the index file.

` import { HomePage , SignIn , SignUp , History, Likes, WatchLater, PlayList } from "./Pages"; `
would work

Reply:
thanks for the suggestion.